.azure-pipelines.yml

```diff
@@ -10,6 +11,27 @@ jobs:
           rust_version: stable
       - bash: |
           export RUST_BACKTRACE=1
-          cargo check --all || exit
           cargo test --all || exit
-        displayName: Run cargo check and test 
+        displayName: Run cargo check and test
```

Comment:
Was there maybe some whitespace changes here?

Reply:
Just removing a trailing space -- I think the issue is `displayName` coming before `script`? Trying to swap the order...